docs/user-guide/commands/argocd_admin_settings_rbac_validate.md

```diff
@@ -10,7 +10,7 @@ a local file, and in either CSV or K8s ConfigMap format.
 
 
 ```
-argocd admin settings rbac validate --policy-file=POLICYFILE [flags]
+argocd admin settings rbac validate --policy-file=POLICYFILE [--namespace=NAMESPACE] [flags]
```

Comment:
Add `--namespace` option under **Options** below. Also, mention somewhere that `policy-file` and `namespace` are mutually exclusive.

Reply:
since the docs are auto generated and the `--namespace` flag is being handled in the following line, you can find the `--namespce` option under **Options inherited from parent commands**.
https://github.com/argoproj/argo-cd/blob/d78ffe624f2da9278d4ded4ab0e9722b4d102016/cmd/argocd/commands/admin/settings_rbac.go#L78